README.md

```diff
@@ -242,27 +242,34 @@ deployments with  `_select`.
    config file:
 
    ```
-   $ mkdir /Users/alice/.gcp
-   $ mv /Users/alice/Downloads/example-project-name_key.json /Users/alice/.gcp/
+   mkdir /Users/alice/.gcp
+   mv /Users/alice/Downloads/example-project-name_key.json /Users/alice/.gcp/
    ```
 
-6. Edit the `environment.local.py` file for your personal deployment and
-   modify the `GOOGLE_APPLICATION_CREDENTIALS` variable:
+6. Edit the `environment.local.py` file for your personal deployment:
 
    ```
    $ vim /Users/alice/azul/deployments/alice.local/environment.local.py
-   
+   ```
+
+   and modify the `GOOGLE_APPLICATION_CREDENTIALS` variable:
+
+   ```
    'GOOGLE_APPLICATION_CREDENTIALS': '/Users/alice/.gcp/example-project-name_key.json'
    ```
 
 7. Repeat the previous step for other deployments as needed or alternatively
    create a symlink to your deployment's `environment.local.py` file:
 
    ```
-   $ cd /Users/alice/azul/deployments/dev
-   $ vim environment.local.py
-     (or)
-   $ ln -snf ../alice.local/environment.local.py environment.local.py
+   cd /Users/alice/azul/deployments/dev
+   vim environment.local.py
+   ```
+
+   or, alternatively:
```

Comment:
This makes it look like the `cd` is only part of the first alternative. Just duplicate it in the second.